ovve_ui/ovve_ui.py

```diff
@@ -104,7 +105,8 @@ def __init__(self) -> None:
         # the MCU.  It will accept settings updates from the UI.
         #
         # When the real comms handler is available, substitute it here.
-        self.comms_handler = CommsSimulator(self.comms_adapter)
+        #self.comms_handler = CommsSimulator(self.comms_adapter)
```

Comment:
We need a command line argument here to allow using the simulator for those testing the UI without Arduino.  I will add it.

Reply:
I merged @BrianWoodruff 's PR which addresses all his comments.